integrations/astra/src/haystack_integrations/document_stores/astra/astra_client.py

```diff
@@ -43,6 +43,19 @@ def __init__(
         similarity_function: str,
         namespace: Optional[str] = None,
     ):
+        """
+        The connection to Astra DB is established and managed through the JSON API.
+        The required credentials (api endpoint and application token) can be generated
+        through the UI by clicking and the connect tab, and then selecting JSON API and
+        Generate Configuration.
+
+        :param api_endpoint: The Astra DB API endpoint.
```

Comment:
for consistency I would start with lower case what comes after a `:param <variable>:`